apps/console/package.json

```
@@ -78,6 +78,7 @@
         "@wso2is/admin.oidc-scopes.v1": "^2.25.204",
         "@wso2is/admin.org-insights.v1": "^2.25.204",
         "@wso2is/admin.organization-discovery.v1": "^2.25.205",
+        "@wso2is/admin.password-recovery-flow-builder.v1": "^0.0.0",
```

shall we use the workspace protocol?

```suggestion
        "@wso2is/admin.password-recovery-flow-builder.v1": "workspace:^",
```